runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -39,10 +40,10 @@
     /**
      * Send a workflow request
      *
-     * @param layout current layout
+     * @param client the client to the selected orchestrator
      * @return a uuid that corresponds to the created workflow
      */
-    protected abstract UUID sendRequest(Layout layout) throws TimeoutException;
+    protected abstract UUID sendRequest(ManagementClient client) throws TimeoutException;
 
     /**
      * Select an orchestrator and return a client. Orchestrator's that
```

Comment:
Can you add a comment saying that we pick the Orchestrator randomly?

Reply:
done.